packages/datatrak-web/src/features/Tasks/StatusPill.tsx

```diff
@@ -46,7 +46,10 @@ export const STATUS_VALUES = {
   },
 };
 
-export const StatusPill = ({ status }: { status: TaskStatusType }) => {
+export const StatusPill = ({ status }: { status?: TaskStatusType | null }) => {
```

Comment:
I'm curious if this was in response to an error that was coming up. It seems to me that the task status should always be supplied.